### PR TITLE
Updated [Term Entry] java string  

### LINKED ISSUE #1588: [Term Entry] Java Strings: `.lastIndexOf()` Method
### Reason/inspiration (optional)

Add a new term entry under the [Java: Strings](https://www.codecademy.com/resources/docs/java/strings) concept. This entry should include:

An explanation of the method.

A "Syntax" section with a ```pseudo code snippet showing the general syntax.

An "Example" section with a ```java code snippet showing how it works.

A markdown file named "lastIndexOf.md" should be added to a folder named "lastIndexOf" and placed in the terms folder located under content/java/concepts/strings/terms

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by jaymay1979**
### Description

Per the review of the new term entry java string file, lastIndexOf.md, the suggested updates and modifications have been made for review. Closes #1588

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+Title: '.lastIndexOf()' 
+Description: 'Searches a string for a specified value, and returns the position of the match.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'String'
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md:7]:
```suggestion
  - 'Strings'

```

File: content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+Title: '.lastIndexOf()' 
+Description: 'Searches a string for a specified value, and returns the position of the match.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'String'
+  - 'Method'
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md:8]:
```suggestion
  - 'Methods'

```

Comment (caupolicandiaz) [reply content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md:8]: if in doubt check our list of tags- https://github.com/Codecademy/docs/blob/main/documentation/tags.md

File: content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Example
+
+This example below searches the string `sentence` for the value `'the'` and returns the index of the last occurrence:
+
+```java
+let str = 'The quick brown fox jumps over the lazy dog';
+let lastPos = str.lastIndexOf('the');
+console.log(lastPos); 
+```
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md:34]:
```java
class Main {
   public static void main(String[] args) {

     String sentence = "The quick brown fox jumps over the lazy dog";

     int lastPos = sentence.lastIndexOf("the");

     System.out.println(lastPos);
  }
}
```

Comment (caupolicandiaz) [reply content/java/concepts/strings/terms/lastIndexOf/lastIndexOf.md:34]: generally we present the examples within the context of a class

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@jaymay1979 Just a few edits, tag me when you've made the updates.

**Review by jaymay1979 — COMMENTED** [review]
I made these final changes on my remote repo, lastIndexOf. Hope my workflow is making since.

**Review by caupolicandiaz — APPROVED** [review]
@jaymay1979 Looks good. This is ready for another review

**Comment by jaymay1979** [comment]
Is there anything on my end that needs to be done to get my next review?

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by yangc95** [comment]
🥳 Thanks for this entry @jaymay1979 here's the live link: https://www.codecademy.com/resources/docs/java/strings/lastIndexOf

Screen Shot 2023-03-02 at 3 44 24 PM

> It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> image
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin

**Pull request merged**